SRC/MISCFUNC.CPP

```diff
@@ -561,9 +564,13 @@ void list_files( int *no, vile *filelist, const char *wildcard, int type )
         }
     }
     globfree(&globp);
+#elif defined(TK_PORT_MSVC)
+    return list_files_msvc( no, filelist, wildcard, type );
 #endif
 }
 
+#ifndef TK_PORT_MSVC
+// windows.h has these already
```

Comment:
Do the `windows.h` versions also modify the argument in-place?

Reply:
Had to try and compare, but yes. Arguments were modified in-place.